src/main/java/by/beloboky/employee/EmployeesAndDutiesFileRepository.java

```diff
@@ -0,0 +1,123 @@
+        }
+    }
+
+    /**
+     * @return The List includes duties.
+     */
+    public List<Duty> readFromFileDuties() {
+        try {
+            return Files.lines(fileDuties.toPath(), StandardCharsets.UTF_8).map(EmployeesAndDutiesFileRepository::convertToDuties).toList();
+        } catch (IOException e) {
+            throw new UncheckedIOException(e);
+        }
+    }
+
+    /**
+     * @param s is string from duties_with_data file.
+     * @return Duties object
+     */
+    private static Duty convertToDuties(String s) {
+        String[] find = s.split(",");
+        LocalDate date = LocalDate.parse(find[1], FORMATTER);
+        int currentID = Integer.parseInt(find[0]);
+        return new Duty(currentID, date, find[2]);
+    }
+
+    /**
+     * @param s is string from data_users file.
+     * @return Employee object
+     */
+    private static Employee convertToEmployee(String s) throws NotFoundEmployeeException {
+        String[] find = s.split(",");
+        int age = Integer.parseInt(find[2]);
+        int salary = Integer.parseInt(find[5]);
+        int workExp = Integer.parseInt(find[6]);
+        int id = Integer.parseInt(find[7]);
+        List<Duty> forEachEmployee = findDutyForEmployee(id);
+        Position position = isEmployeePosition(find);
+        Sex sex = isEmployeeSex(find);
+        return new Employee(find[0], find[1], age, sex, position, salary, workExp, forEachEmployee, id);
+    }
+
+    /**
+     * @param find - take a string with employee's position.
+     * @return Position type, what the employee's position is.
+     */
+    private static Position isEmployeePosition(String[] find) throws NotFoundEmployeeException {
+        if (Objects.equals(find[4], "manager")) {
+            return Position.MANAGER;
+        } else if (Objects.equals(find[4], "employee")) {
+            return Position.EMPLOYEE;
+        } else if (Objects.equals(find[4], "security")) {
+            return Position.SECURITY;
+        } else if (Objects.equals(find[4], "director")) {
+            return Position.DIRECTOR;
+        } else {
+            throw new NotFoundEmployeeException("Not found position");
+        }
+    }
+
+    /**
+     * @param find - take a string with employee's sex.
+     * @return Sex that the employee has.
+     */
+    private static Sex isEmployeeSex(String[] find) throws NotFoundEmployeeException {
+        if (Objects.equals(find[3], "M") || Objects.equals(find[3], "F")) {
+            return Objects.equals(find[3], "M") ? Sex.MALE : Sex.FEMALE;
+        } else {
+            throw new NotFoundEmployeeException("Not found position");
+        }
+    }
+
+    /**
+     * @param id - id employee;
+     * @return List Duty type of each employees;
+     */
+    private static List<Duty> findDutyForEmployee(int id) {
+        List<Duty> forEachEmployee = new LinkedList<>();
+        EmployeesAndDutiesFileRepository takeDutyList = new EmployeesAndDutiesFileRepository();
```

Comment:
`EmployeesAndDutiesFileRepository takeDutyList = new EmployeesAndDutiesFileRepository();`

Why did you create an repository in the repository?